app/views/admin/complete_persons.html.erb

```diff
@@ -1,25 +1,23 @@
 <% provide(:title, 'Finish unfinished persons') %>
 
 <div class="container">
-  <% competition_ids = @finish_persons.competition_ids %>
-
   <h2>
     <%= yield(:title) %>
-    <%= link_to "Go back", admin_finish_unfinished_persons_path(competition_ids: competition_ids), class: "btn btn-default" %>
+    <%= link_to "Go back", panel_page_path(id: User.panel_pages[:createNewComers], competition_ids: @competition_ids.join(',')), class: "btn btn-default" %>
```

Comment:
In regards to my other comment above: Kindly pointing out that here, you're fine with comma-glueing, but above in the other URL helper you insist on array-wrapping :wink: